frontend/src/components/AdminControls/Users.css

```diff
@@ -0,0 +1,48 @@
+.button {
```

Comment:
Is this a copy of the css from the other file? We should only need to define it once.